src/widgetastic_patternfly/__init__.py

```diff
@@ -229,6 +229,14 @@ def help_block(self):
         else:
             return self.browser.text(help_block)
 
+    @property
+    def warning(self):
+        try:
+            self.browser.wait_for_element('./following-sibling::div', timeout=3)
```

Comment:
Let's store this locator in `Input.WARNING_LOC` so it can be easily overriden and modified.